packages/material-ui/src/FormControlLabel/FormControlLabel.test.js

```diff
@@ -30,126 +35,141 @@ describe('<FormControlLabel />', () => {
   }));
 
   it('should render the label text inside an additional element', () => {
-    const wrapper = findOutermostIntrinsic(
-      mount(<FormControlLabel label="Pizza" control={<div />} />),
-    );
-    assert.strictEqual(wrapper.name(), 'label');
-    assert.strictEqual(wrapper.text(), 'Pizza');
-    assert.strictEqual(wrapper.hasClass(classes.root), true);
+    const { container, getByText } = render(<FormControlLabel label="Pizza" control={<div />} />);
+    const root = container.firstChild;
```

Comment:
Should we extend our clientRender to expose `root`? It seems to be a common pattern. I'm not expecting an answer now, but it could be a question we keep in the back of our head.

Reply:
I'm not even sure we have a single component that exposes two DOM nodes by default, nor that it should be encouraged. Maybe we could merge root and container as a forcing function to prevent it.

Reply:
I was thinking the same. Basically as a replacement for `findOutermostIntrinsic`. But I'm not sure yet if it's really that important since it's just an alias for `firstChild`. Let's see how this develops.